versioned_docs/version-v1.1/rancher/node/node-driver.md

```diff
@@ -31,6 +31,55 @@ When the Harvester node driver is enabled, you can create Kubernetes clusters on
 ### Support Matrix
 Refer to [Rancher Downstream Cluster Support Matrix](https://www.suse.com/suse-rancher/support-matrix/all-supported-versions/rancher-v2-6-9)
 
+## Known issues
+
+| Summary                                                                                                                                                                                   | Status    | Last updated |
+|-------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|-----------|--------------|
+| [Volumes created by the Harvester CSI driver in the host Harvester cluster would be deleted after editing/deleting the guest cluster](https://github.com/harvester/harvester/issues/3272) | Mitigated | 2023-03-15   |
+
+## Issue details
+
+### Volumes created by the Harvester CSI driver in the host Harvester cluster would be deleted after editing/deleting the guest cluster.
+| Status    | Last updated |
+|-----------|--------------|
+| Mitigated | 2023-03-15   |
+
+**Workaround**: Temporary change the Harvester node driver version to [v0.6.3](https://github.com/harvester/docker-machine-driver-harvester/releases/tag/v0.6.3) in Rancher can help. This can be accomplished via the Rancher UI.
+1. Click `View in API` in Rancher `Cluster Management` > `Drivers` > `Node Drivers` > `Harvester` > `⋮`
+2. Click `Edit`
+3. Uncheck the `builtin` checkbox
+4. Change the `*url` to `https://releases.rancher.com/harvester-node-driver/v0.6.3/docker-machine-driver-harvester-amd64.tar.gz`
+5. Change the `checksum` to `159516f8f438e9b1726418ec8608625384aba1857bc89dff4a6ff16b31357c28`
+6. Click `Show Request` > `Send Request`
```

Comment:
It would be nice to add a step to verify the node driver is loaded correct.

Not sure if this is good enough:
![Screenshot 2023-03-15 at 3 02 35 PM](https://user-images.githubusercontent.com/1691518/225231758-55690071-2dea-4294-9769-d6633f42f644.png)

Reply:
Added the step 7: Click `Reload` util the value of `status.appliedChecksum` and `status.appliedURL` change to the value we set.